hack/testdata/kubernetes-service.yaml

```diff
@@ -4,7 +4,6 @@ metadata:
   name: kubernetes
   namespace: default
 spec:
-  clusterIP: 10.0.0.1
```

Comment:
fixed a different way (try to create, tolerate already existing, then verify existence)